src/pages/Search/SearchResultPage.tsx

```
@@ -1,74 +1,154 @@
-import type { TroubleShootingCardProps } from "@/components/MyPage/TroubleShootingCard";
 import TroubleShootingCard from "@/components/MyPage/TroubleShootingCard";
-import { useEffect, useState } from "react";
+import { useInfiniteCommunityTroubleSearch } from "@/hooks/useInfiniteCommunityTroubleSearch";
+import { useInfiniteMyTroubleSearch } from "@/hooks/useInfiniteMyTroubleSearch";
+import { useInfiniteUserTroubleSearch } from "@/hooks/useInfiniteUserTroubleSearch";
+import { useEffect, useRef } from "react";
 import { useLocation } from "react-router-dom";
 
 type CardData = Omit<TroubleShootingCardProps, "onDeleted">;
```

ESLint

[error] 'CardData' is defined but never used.

(@typescript-eslint/no-unused-vars)